### ISSUE TITLE
[TODO] Create all global style files

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/RadioLogger#27]
We have global styles availale to us by by FontAwesome 5 and Bootstrap 5.

We have app.css, index.css.  

We need to create global stying that will be pushed through the whole entire app.

Ideas?

**Comment by jzunigarce** [comment]
We can separate the source in pages(folders) and layouts(head, fotter, etc). We have configure node for work with templates

**Comment by gbowne1** [comment]
Yes.  I prefer global styles.  Then issuing styles for anything a particular page needs individual more granular stying on top of that  We do not need to over duplicate styling which may confuse the browser.  

Then we can move towards scss/sass and install the sass tooling.  Its a lot cleaner.

**Comment by gbowne1** [comment]
I am all about reusable components.
